bundle/config/resources/job.go

```diff
@@ -15,6 +16,16 @@ type Job struct {
 	*jobs.JobSettings
 }
 
+func (s *Job) UnmarshalJSON(b []byte) error {
+	type C Job
+	return marshal.Unmarshal(b, (*C)(s))
```

Comment:
Why to do this conversion if `s` is already of type `*Job`?

Reply:
It is not necessary anymore, but for future reference:
Until recent changes, our custom Marshaler would use `json.Marshal/Unmarshal` under the hood and then do some tweaks on top. 
That would cause a infinite loop of `encoding/json` calling this function again. By creating this new `C` type, it masks this method and then `encoding/json` would do the normal encoding. 
We don't use `json.Marshal/Unmarshal` internally anymore in this way, so I am removing this.